### PR TITLE
Fixes #14510 - Hammer uses /config_reports rather than /reports

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

**Comment by tstrachota** [comment]
Thanks @ofedoren. It all looks sane to me but I need to test it. If there's any volunteer who could do the testing, I'd appreciate it.

**Comment by mbacovsky** [comment]
@tstrachota I'll do the testing

**Comment by tstrachota** [comment]
Thanks @mbacovsky Feel free to merge if it works fine.

File: lib/hammer_cli_foreman/host.rb [hunk 1]
```diff
@@ -213,15 +213,14 @@ def send_request
       end
     end
 
+    class ConfigReportsCommand < HammerCLIForeman::ListCommand
+      command_name 'config-reports'
```
Comment (tstrachota) [inline lib/hammer_cli_foreman/host.rb]: Hmm, we should probably keep the `reports` command with a deprecation warning for the sake of compatibility.

File: lib/hammer_cli_foreman/config_report.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 module HammerCLIForeman
 
-  class Report < HammerCLIForeman::Command
+  class ConfigReport < HammerCLIForeman::Command
```
Comment (mbacovsky) [inline lib/hammer_cli_foreman/config_report.rb]: @ofedoren, there is still reference to `Report` in hammer_cli_foreman.rb L105. The "global" report command there should be also renamed. 

I agree with Tomas's point about keeping the report command as deprecated for a version or two.

**Comment by ofedoren** [comment]
@mbacovsky, @tstrachota thanks!

Updated: Both `report` and `host reports` commands have deprecation warning now.

**Comment by mbacovsky** [comment]
:+1: Well done, thanks @ofedoren!

**Pull request merged**